pyhive/sqlalchemy_presto.py

```diff
@@ -80,6 +89,7 @@ class PrestoDialect(default.DefaultDialect):
     supports_multivalues_insert = True
     supports_unicode_statements = True
     supports_unicode_binds = True
+    supports_statement_cache = False
```

Comment:
Added this attribute to disable the warning related to SQL compilation caching. The attribute should only be enabled once the dialect has been altered as needed and tested for reusability of compiled SQL statements with differing parameters, hence set it to False as this is out of scope for this PR.  

https://docs.sqlalchemy.org/en/14/errors.html#error-cprf
https://docs.sqlalchemy.org/en/14/core/connections.html#sql-compilation-caching
https://docs.sqlalchemy.org/en/14/core/connections.html#engine-thirdparty-caching